src/providers/analytics.ts

```diff
@@ -36,8 +36,12 @@ export const provider = 'google.analytics';
 export const service = 'app-measurement.com';
 
 /**
- * Select analytics events to listen to for events.
- * @param analyticsEventType Name of the analytics event type.
+ * Registers a Cloud Function to handle analytics events.
```

Comment:
We're inconsistent on this unfortunately, but wherever you see "Cloud Function" used to refer to an actual single, discrete function, replace it with "function."